test/e2e-tests.sh

```diff
@@ -49,11 +49,11 @@ use_https=""
 (( MESH )) && parallelism="-parallel 1"
 
 if (( HTTPS )); then
-  parallelism="-parallel 1"
   use_https="--https"
   turn_on_auto_tls
   kubectl apply -f ./test/config/autotls/certmanager/caissuer/
   add_trap "kubectl delete -f ./test/config/autotls/certmanager/caissuer/ --ignore-not-found" SIGKILL SIGTERM SIGQUIT
+  add_trap "turn_off_auto_tls" SIGKILL SIGTERM SIGQUIT
```

Comment:
This change is suggested by https://github.com/knative/serving/pull/7255#discussion_r396634418. It is not related to this PR but small change, so I included it in this PR.